pyproject.toml

```diff
@@ -1,3 +1,35 @@
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools >= 36.6"]
+requires = ["setuptools >= 61.0"]
```

Comment:
I'd be curious about which part needs this bump.

Reply:
Setuptools v61 is the first version which supports project metadata in project.toml